core/src/main/java/org/apache/spark/shuffle/sort/UnsafeShuffleWriter.java

```diff
@@ -321,11 +334,14 @@ void forceSorterToSpill() throws IOException {
   }
 
   /**
-   * Merges spill files using Java FileStreams. This code path is slower than the NIO-based merge,
-   * {@link UnsafeShuffleWriter#mergeSpillsWithTransferTo(SpillInfo[], File)}, so it's only used in
+   * Merges spill files using Java FileStreams. This code path is typically slower than the NIO-based merge,
```

Comment:
nit: some of these lines are great than 100ch (in comments and code). Can you please fix those?